### DISCUSSION TITLE
Very stranger light and semi transparency

### DISCUSSION THREAD

**Discussion by punkesta** [sceneview/sceneview#498] [Q&A]
Does anyone know how to fix this transparency on the model?

https://github.com/SceneView/sceneview-android/assets/99623192/01fdb57b-3364-496b-b6b7-9f7a4c619373

**Comment by ThomasGorisse** [comment]
Strange lighting and semi-transparency artifacts are usually caused by one of:

1. **Missing environment HDR** — Without an environment, models appear dark or have incorrect reflections. Make sure you load an HDR environment.
2. **Material blending mode** — If your GLB uses opaque materials but has alpha in textures, you'll get artifacts. Set materials to BLEND mode in your 3D authoring tool.
3. **Double-sided rendering** — Single-sided faces can cause transparency-like artifacts. Enable double-sided in your material.

In v4.0.0-rc.1, the default environment lighting should handle most cases correctly. If you're still seeing issues, try loading a known-good model like the DamagedHelmet from the Khronos glTF samples to isolate whether the issue is in your model or the setup.
